docs/usage/guides/calico.md

```diff
@@ -24,22 +10,33 @@ And then you have to change the ConfigMap `calico-config`. On the `cni_network_c
         "allow_ip_forwarding": true
     }
 ```
-Or directly using the descriptor calico.yaml
-
+Or you can directly use this [calico.yaml](calico.yaml) manifest
 
-### 2.2. Apply the descriptor
-With kubectl (check the usage `kubectl kubeconfig` if you can't connect to the cluster).
+## 2. Create the cluster without flannel and with calico
+On the k3s cluster creation :
+- add the flag `--flannel-backend=none`. For this, on k3d you need to forward this flag to k3s with the option `--k3s-server-arg`.
+- mount (`--volume`) the calico descriptor in the auto deploy manifest directory of k3s `/var/lib/rancher/k3s/server/manifests/`
 
-From the root of the repository
+So the command of the cluster creation is (when you are at root of the k3d repository)
 ```bash
-    kubectl apply -f docs/usage/guides/calico.yaml
+    k3d cluster create "${clustername}" --k3s-server-arg '--flannel-backend=none' --volume "$(pwd)/docs/usage/guides/calico.yaml:/var/lib/rancher/k3s/server/manifests/calico.yaml"
 ```
+In this example :
+- change `"${clustername}"` with the name of the cluster (or set a variable). 
+- `$(pwd)/docs/usage/guides/calico.yaml` is the absolute path of the calico manifest, you can adapt it.
+
+You can add other options, [see](../commands.md).  
 
-And watch for the calico pod(s) deployment
+The cluster will start without flannel (and without other CNI).
```

Comment:
Well, in this variant, it will start with Calico as the CNI Plugin, right?

Reply:
yes, I will had a capture of this

Reply:
update done @iwilltry42